packages/jest-snapshot/src/utils.js

```diff
@@ -179,3 +187,59 @@ export const saveSnapshotFile = (
     writeSnapshotVersion() + '\n\n' + snapshots.join('\n\n') + '\n',
   );
 };
+
+export const saveInlineSnapshots = (
+  snapshotData: {[key: string]: InlineSnapshot},
+  sourceFilePath: Path,
+) => {
+  const sourceFile = fs.readFileSync(sourceFilePath, 'utf8');
```

Comment:
We be awesome to reuse the hastemap to save the fs operation.

Then we should be able to cache prettier config lookup as well

Reply:
I'm unfamiliar with hastemap, got any pointers to how I would do that?

Reply:
I'll have to take a look when I'm at a computer, but basically the context contains the full filesystem (and dependencies between files) in an abstraction called haste map (with haste fs in it). I'm not sure how to best access it (or if it's even possible) in jest-snapshot